### ISSUE TITLE
Unexplained charges using chat app

### ISSUE THREAD

**Issue by Darth-Coin** [stakwork/sphinx-relay#196]
I was testing with a friend the sphinx chat tribes.
So he created a tribe, I joined, all OK.
We both have a mutual channel open between our nodes.
We both have a channel open with gameb_1 Sphinx node. We have enough balanced channels for in/out.
The fee to chat in the tribe is 0 sats. Joining is 1 sat (just for testing).

But I notice a strange and unexplained (anywhere) behaviour:
- each action in sphinx app on android is charging 3 sats, anything you do there is 3sats. I have like 12 charges each 3 sats
- I have only 1 returned 3 sats.

For what are those charges? And why the user is NOT informed about it? If this continue like that, it is worse than a bank that rob your account.
I understand that have to be some fees for chating etc. nothing is free, we know that. But at least inform the user BEFORE: hey you will pay for this and this. Did you followed the steps? Do you agree?
But there's no informed consent about this.

**Comment by Evanfeenstra** [comment]
Yes, messages sent in a tribe cost 3 sats (or more, depending on the chat admin's settings). We can work on informing better in the app. Also, we will work on making these fees lower and lower over time

**Comment by Darth-Coin** [comment]
@Evanfeenstra don't get me wrong... I am not against the fees, everybody have to pay his price.
But for changing my profile picture I get charged 3 sats, by my tribe peer?
For changing some settings in chat app I am charged 3 sats also by my peer?
What have to do a tribe with changing my picture on chat app?
Something doesn't add up here...

**Comment by Evanfeenstra** [comment]
@Darth-Coin when you change your profile pic you send a keysend to your friends letting them know it has changed. I agree, we need to figure out a better way to do this! 

Please let me know if you have an idea how to do this in a decentralized way (no central server where profile pics are "registered")
